<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-ADUL7BM" failures="1" tests="6" name="Default test(failed)" time="59.654" errors="0" timestamp="12 Aug 2024 12:26:44 GMT">
  <testcase classname="Practice.Test.InvoiceTest2" name="createInvoiceTest2" time="0.173">
    <skipped/>
  </testcase> <!-- createInvoiceTest2 -->
  <testcase classname="Practice.Test.InvoiceTest2" name="createInvoiceTest2" time="0.034">
    <skipped/>
  </testcase> <!-- createInvoiceTest2 -->
  <testcase classname="Practice.Test.InvoiceTest2" name="createInvoiceTest2" time="0.051">
    <skipped/>
  </testcase> <!-- createInvoiceTest2 -->
  <testcase classname="Practice.Test.InvoiceTest2" name="createInvoiceTest2" time="0.029">
    <skipped/>
  </testcase> <!-- createInvoiceTest2 -->
  <testcase classname="Practice.Test.InvoiceTest2" name="createInvoiceTest2" time="0.046">
    <skipped/>
  </testcase> <!-- createInvoiceTest2 -->
  <testcase classname="Practice.Test.InvoiceTest2" name="createInvoiceTest2" time="0.044">
    <failure type="java.lang.AssertionError" message="expected [Login] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]">
      <![CDATA[java.lang.AssertionError: expected [Login] but found [Administrator - Home - vtiger CRM 5 - Commercial Open Source CRM]
at Practice.Test.InvoiceTest2.createInvoiceTest2(InvoiceTest2.java:18)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- createInvoiceTest2 -->
  <testcase classname="Practice.Test.InvoiceTest2" name="createInvoicewithConatctTest" time="0.0">
    <ignored/>
  </testcase> <!-- createInvoicewithConatctTest -->
</testsuite> <!-- Default test(failed) -->
